hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/consumer/sender/MultiMessageSender.java

```diff
@@ -0,0 +1,27 @@
+package pl.allegro.tech.hermes.consumers.consumer.sender;
+
+import pl.allegro.tech.hermes.consumers.consumer.Message;
+import pl.allegro.tech.hermes.consumers.consumer.SendFutureProvider;
+
+import java.util.concurrent.CompletableFuture;
+
+public abstract class MultiMessageSender implements MessageSender {
```

Comment:
I'm thinking about more verbose/precise name such as `MultiRecipientMessageSender` ?

Reply:
On the other hand, do we need this abstract class ? It has only one implementation and almost no specific logic :)

Reply:
agree, removed it. It can be added once there are more multi recipient message senders